mlflow/__init__.py

```diff
@@ -1,22 +1,43 @@
+"""Provides the MLflow fluent API, allowing management of an active MLflow run.
+For example::
```

Comment:
Using `..code::` instead of `::` will make it look better in the generated docs (for instance it adds the "copy this code" button and syntax highlighting).